metabase/metabase
# Add markdown label for events

Epic: https://github.com/metabase/metabase/issues/20289

How to test:
- Go to `/collection/root/timelines/new/events/new`
- Make sure there is `Markdown supported` label

<img width="663" alt="Screenshot 2022-03-25 at 19 44 19" src="https://user-images.githubusercontent.com/8542534/160164359-05833938-b5a9-43e6-9bf6-259efca05a63.png">
<img width="662" alt="Screenshot 2022-03-25 at 19 44 24" src="https://user-images.githubusercontent.com/8542534/160164367-a2722918-0c51-4f8a-9fe4-0c9f7b06f413.png">
